package.json

```diff
@@ -14,11 +14,11 @@
   "author": "Blackbaud",
   "license": "MIT",
   "dependencies": {
-    "@blackbaud/skyux": "2.0.0-beta.26",
+    "@blackbaud/skyux": "2.0.0-beta.29",
     "@blackbaud/stache": "2.0.0-beta.0"
```

Comment:
Once we've updated skyux and builder versions on stache (and added the plugin to package.json), we can update the version here to `2.0.0-beta.1`